src/containers/OLDrawFeatures/OLDrawFeatures.js

```
@@ -0,0 +1,288 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { VISUALIZE_DRAW_MODES } from './../../constants';
+import { Draw, Modify, Select, Snap } from 'ol/interaction';
+import Map from 'ol/Map';
+import { GeoJSON, WMSCapabilities } from 'ol/format';
+import { createRegularPolygon, createBox } from 'ol/interaction/Draw';
+import { platformModifierKeyOnly, altKeyOnly, shiftKeyOnly, singleClick, doubleClick } from 'ol/events/condition';
+import { Vector as VectorSource } from 'ol/source';
+import { Vector as VectorLayer } from 'ol/layer';
+import { Circle, Fill, Text, Stroke, Style, RegularShape } from 'ol/style';
+import { actions as layerCustomFeatureActions } from '../../redux/modules/LayerCustomFeature';
+
+const INDEX_BOUNDING_BOX = 101;
+const LAYER_BOUNDING_BOX = 'LAYER_BOUNDING_BOX';
+
+// Interesting try/catch to avoid main thread error (on Polygon/LineString drawing with Select/Modify interactions)
+// https://github.com/openlayers/openlayers/issues/6310
+import RBush from 'ol/structs/RBush';
+const RBushUpdateOrig = RBush.prototype.update;
+RBush.prototype.update = function () {
+  try {
+    RBushUpdateOrig.apply(this, arguments);
+  } catch (e) {
+    // console.warn('RBushUpdateOrig', e);
```

Est-ce qu'il y avait une notification à montrer au user ici? Est-ce que c'est en plan ou on veut vraiment swallow l'exception?

---

Pas de notification au user, mais je vais laisser le warning dans la console